autobuild/build.sh

```diff
@@ -214,12 +214,9 @@ cd synfig-core
 pushd "${REPO_DIR}/synfig-core/" >/dev/null
 /bin/bash "${REPO_DIR}/synfig-core/bootstrap.sh"
 popd >/dev/null
-if [ -e /etc/debian_version ] && [ -z "$BOOST_CONFIGURE_OPTIONS" ]; then
+if [ -e /etc/debian_version ]; then
 	# Debian/Ubuntu multiarch
 	MULTIARCH_LIBDIR="/usr/lib/`uname -m`-linux-gnu/"
```

Comment:
Is `MULTIARCH_LIBDIR` still needed after this PR?

Reply:
`MULTIARCH_LIBDIR` not needed anymore :+1: